### DISCUSSION TITLE
Strange blink (repo octocat)

### DISCUSSION THREAD

**Discussion by Chimildic** [docsifyjs/docsify#1605] [Q&A] [answered]
Sometimes when click to change theme or just new page on sidebar, shows strange page for 0.5 second and then normal page. You can try [on site](https://chimildic.github.io/goofy/#/func). 
What is it, how to fix?

## Screenshot

![2021-06-28_12-11-25](https://user-images.githubusercontent.com/40138097/123671095-9429a680-d846-11eb-82cd-2869fb1bac21.png)

**Comment by sy-records** [comment]
This seems to be caused by incompatibility between the theme and the plugin, you can exclude

**Comment by Chimildic** [comment, marked as answer]
There was too heavy style. Both theme light and dark were nearly the same. The blink was when switch from one to another. Including switch pages. 
I extracted equal styles in `base-theme.css` and difference to `light-theme.css` and `dark-theme.css`. The blink gone.
